gallery/visualtests/nxCombobox.js

```diff
@@ -232,17 +232,44 @@ describe('NxCombobox', function() {
 
   it('passes a11y checks', a11yTest());
 
-  it('shows the dropdown when focused and hides the dropdown when not focused', async function() {
-    const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
-        dropdownMenuSelector = `${basicExampleSelector} .nx-dropdown-menu`,
-        [input, dropdownMenu] = await waitAndGetElements(inputSelector, dropdownMenuSelector);
+  describe('dropdown display behavior', function() {
+    it('shows the dropdown when initially focused', async function() {
```

Comment:
This is only true in the case where the list of options is immediately available.  In the case where the options populate once you start typing, the dropdown _should not_ be visible until you start typing.  I wonder if this entire `describe` block should be duplicated, once for the "Basic Example" and once for the "Backend Query Example", with the first test in the latter changed to check that the dropdown opens once the user has typed.

Reply:
> This is only true in the case where the list of options is immediately available. In the case where the options populate once you start typing, the dropdown _should not_ be visible until you start typing. I wonder if this entire `describe` block should be duplicated, once for the "Basic Example" and once for the "Backend Query Example", with the first test in the latter changed to check that the dropdown opens once the user has typed.

Makes sense! Added here: [f5745f9](https://github.com/sonatype/sonatype-react-shared-components/pull/876/commits/f5745f9ad1c5136dc159b4eac0c839e66ac53348)